source/fuzz/fuzzer_context.cpp

```diff
@@ -61,6 +61,7 @@ const std::pair<uint32_t, uint32_t> kChanceOfChoosingStructTypeVsArrayType = {
 const std::pair<uint32_t, uint32_t> kChanceOfChoosingWorkgroupStorageClass = {
     50, 50};
 const std::pair<uint32_t, uint32_t> kChanceOfConstructingComposite = {20, 50};
+const std::pair<uint32_t, uint32_t> kChanceOfPropagatingInstructions = {20, 50};
```

Comment:
I would be inclined to up the upper bound to 70 here.

Reply:
Fixed in c74f0e3.